private/private.h

```diff
@@ -191,6 +191,8 @@ void _dispatch_prohibit_transition_to_multithreaded(bool prohibit);
 typedef mach_port_t dispatch_runloop_handle_t;
 #elif defined(__linux__) || defined(__FreeBSD__)
 typedef int dispatch_runloop_handle_t;
+#elif defined(__unix__)
+typedef uint64_t dispatch_runloop_handle_t;
```

Comment:
This feels a bit ... worrisome to me.  If this ever gets re-ordered, we will have a subtle way to change the semantics.  Currently `__linux__` will define `dispatch_runloop_handle_t` to `int` rather than `uint64_t`.  Is there a reason to prefer the `__unix__` spelling here over `__OpenBSD__`?